homework-003/taskboard.js

```diff
@@ -0,0 +1,121 @@
+class Task {
+  constructor(task, name){
+    this.task = task;
+    this.name = name;
+  }
+
+  render(){
+    if(this.name){
+      return `${this.task} • ${this.name}`;
+    }
+    else {
+      return `${this.task}`;
+    }
+  }
+}
+
+class List {
+  constructor(list){
+    this.list = list;
+    this.arr = [];
+  }
+
+  addTask(task){
+    this.arr.push(task);
+    return this;
+  }
+
+  removeTask(job){
+    for(let i = 0; i < this.arr.length; i++){
+      if(this.arr[i].task === job){
+        return this.arr.splice(i, 1)[0];
+      }
+    }
+  }
+
+  render(){
+    let listing = [];
+    listing.push(`|--------------\n| ${this.list}\n|--------------`)
+    for(const items of this.arr){
```

Comment:
similarly, a better name than `items` is `task`

The difference may seem unimportant now, but as code grows larger being able to understand the intention of a line without having to scroll back to see where a variable was created becomes an important productivity issue.